README.md

```diff
@@ -326,7 +330,17 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
     * [Code Newbie](https://www.codenewbie.org/podcast)Stories from people on their coding journey.
     * [Ladybug Podcast](https://ladybug.dev/) An all lady-hosted tech podcast for all developers.
 
+* Spanish
+    
+    - [Aws by FooBar](https://www.youtube.com/watch?v=94etXBFDDh8)
+    - [No Another AWS podcasts](https://www.youtube.com/watch?v=W6NyjNUC_uY)
```

Comment:
We encourage using channel links instead of videos'

```suggestion
   - [Aws by FooBar](https://www.youtube.com/channel/UCSLIvjWJwLRQze9Pn4cectQ)
    - [No Another AWS podcasts](https://www.youtube.com/channel/UCZPiAjdoHhVNU7h9xnanGkw)
```